### ISSUE TITLE
Lsp doesn't work correctly 

### ISSUE THREAD

**Issue by AdiCahyaSaputra** [LintaoAmons/cd-project.nvim#16]
https://github.com/LintaoAmons/cd-project.nvim/assets/77385046/68422694-530d-4d2e-a978-21e3432f8d43

it happen when trying to cd into another project while my lsp start in the first opened project

**Comment by LintaoAmons** [comment]
Sorry, but this plugin did nothing with lsp, so I'm also not very sure what was going on there...
Maybe the `cwd` changes caused the trouble?

**Comment by AdiCahyaSaputra** [comment]
> Sorry, but this plugin did nothing with lsp, so I'm also not very sure what was going on there... Maybe the `cwd` changes caused the trouble?

yes, but with different LSP (e.g project 1 use intelephense, project 2 use tsserver) it works

**Comment by AdiCahyaSaputra** [comment]
The problem can be resolved by Restarting LSP server using `:LspRestart`. If i run this on hooks callback like this
```lua
  hooks = {
    {
      callback = function(_)
        if next(vim.lsp.get_active_clients()) ~= nil then
          print "Restarting LSP..."
          vim.cmd "LspRestart"
        end
      end,
    },
  },
```

But the command `LspRestart` not restarting LSP server unless i do it manually `:LspRestart`

**Comment by AdiCahyaSaputra** [comment]
i thinks its work for now 
```lua
  hooks = {
    {
      callback = function(_)
        if next(vim.lsp.get_active_clients()) ~= nil then
          vim.cmd "%bd!"
          vim.cmd "LspRestart"
        end
      end,
    },
  },

```
